pom.xml

```diff
@@ -169,36 +200,6 @@
 
     <pluginRepositories>
         <pluginRepository>
-            <releases />
```

Comment:
I don't know enough about Maven to know if these are breaking changes or not -- I'd really like someone who does to have a second look

Reply:
I am not a maven expert too, but I think that the tags below defines if is allowed to use snapshots and releases.  I think the default is enable to releases and false to snapshots.

         <releases>
                <enabled>true</enabled>
            </releases>
            <snapshots>
                <enabled>false</enabled>
            </snapshots>